src/pages/responsible-trading/_trading-responsibly.js

```diff
@@ -62,30 +62,34 @@ const ContentWrapper = styled(Flex)`
         flex-direction: column;
     }
 `
-// const StyledFlex = styled(Flex)`
-//     max-width: 58rem;
-//     max-height: 6.6rem;
-//     border-radius: 6rem;
-//     box-shadow: inset 0 -1px 0 0 #f2f3f4;
-//     background-color: var(--color-grey-25);
-//     margin-left: auto;
-//     margin-right: auto;
-//     margin-top: 4rem;
-//     flex-direction: row;
-//     justify-content: center;
-//     align-items: center;
+const StyledFlex = styled(Flex)`
```

Comment:
how about if you add `padding` for desktop version

Reply:
Sure, will be incorporated